include/ofi.h

```diff
@@ -158,16 +158,16 @@ static inline int ofi_val32_ge(uint32_t x, uint32_t y) {
 #define TAB "    "
 
 #define CASEENUMSTR(SYM) \
-	case SYM: { ofi_strcatf(buf, #SYM); break; }
+	case SYM: { ofi_strcatf(buf, buflen, #SYM); break; }
```

Comment:
Rather than changing ofi_strcatf to accept len, it seems better to replace the call with ofi_strncatf() everywhere len needs to be checked.  This could result in removing ofi_strcatf completely, if it no longer ends up being called.

Reply:
Done. I updated the code and removed `ofi_strcatf` completely.